XouDevSpec/Classes/features/Apis/XouBase/Urls/Urls.swift

```diff
@@ -8,7 +8,9 @@ import Foundation
 
 public struct UrlStrings {
     public let getUsersUrl = "https://xoubase.herokuapp.com/users"
-    public let getUsersLocalUrl = "http://localhost:8080"
+    public let getUsersLocalUrl = "http://localhost:8080/users"
+    public let registerUserUrl = "http://localhost:8080/send"
+    public let loginUserUrl = "http://localhost:8080/authenticate"
```

Comment:
If you're gonna host on Heroku then localhost won't work anymore. You'll have to get the proper url that's based on heroku. It would be something like "xoubase.heroku.com/users"

Reply:
I'll change it now = ) (It slipped my mind)